components/nimbus/src/lib.rs

```diff
@@ -658,18 +88,19 @@ pub struct Experiment {
     // but we ignore them because they're for internal use by RemoteSettings.
 }
 
+#[allow(unused)]
```

Comment:
You'll see a number of these around now, they're required because the `rust-clippy` tests run against both `--no-default-features` and with features. When the `--no-default-features` flag is used, a lot of the stuff in `lib` and other places doesn't get used, resulting in the `rust-clippy` job failing. 

As we build out the Cirrus feature we should be able to remove these (hopefully).

Reply:
One thing I'm curious about that I've not been able to find is if there's a way to group attributes, or have attributes apply to attributes. 

As an example, what I'd like to do here is something like 
```rust
#[allow(unused), cfg(not(feature = "nimbus"))]
```
to note allow unused only when the `nimbus` feature is disabled (this exact syntax is definitely not supported, and the only things related to something like it I've found were closed out and not implemented).

Reply:
may be doing this` #[cfg_attr(not(feature = "nimbus"), allow(unused))]`
this allow(unused) attribute only when the nimbus feature is disabled. When the nimbus feature is enabled, the allow(unused) attribute will not be applied.

Reply:
AHH That's what I was trying to find, thank you! I was expecting there to simply be an option to join cfg attributes together, but `cfg_attr` seems to be specifically for this exact purpose. Great find!!